Fails to build for me on macos 10.15:

```
  CCLD     fuzz_format_parsers
ld: unknown option: --wrap=strdup
clang-7: error: linker command failed with exit code 1 (use -v to see invocation)
make[1]: *** [Makefile:409: fuzz_format_parsers] Error 1
make[1]: Leaving directory '/private/tmp/nix-build-pam_u2f-1.1.1.drv-0/pam_u2f-1.1.1/fuzz'
make: *** [Makefile:756: all-recursive] Error 1
builder for '/nix/store/ngbf652wbpr6nb5h9wl936irvf5gmkzw-pam_u2f-1.1.1.drv' failed with exit code 2
error: build of '/nix/store/ngbf652wbpr6nb5h9wl936irvf5gmkzw-pam_u2f-1.1.1.drv' failed
```

--- a/pkgs/os-specific/linux/pam_u2f/default.nix
+++ b/pkgs/os-specific/linux/pam_u2f/default.nix
@@ -2,11 +2,11 @@
 
 stdenv.mkDerivation rec {
   pname = "pam_u2f";
-  version = "1.1.0";
+  version = "1.1.1";
 
   src     = fetchurl {
     url = "https://developers.yubico.com/pam-u2f/Releases/${pname}-${version}.tar.gz";
-    sha256 = "01fwbrfnjkv93vvqm54jywdcxa1p7d4r32azicwnx75nxfbbzhqd";
+    sha256 = "12p3pkrp32vzpg7707cgx8zgvgj8iqwhy39sm761k7plqi027mmp";
   };
 
   nativeBuildInputs = [ pkg-config ];
@@ -14,6 +14,14 @@ stdenv.mkDerivation rec {
 
   preConfigure = ''
     configureFlagsArray+=("--with-pam-dir=$out/lib/security")
+  '';
+
+  # a no-op makefile to prevent building the fuzz targets
+  postConfigure = ''
+    cat > fuzz/Makefile <<EOF
+    all:
+    install:
+    EOF
   '';
 
   meta = with lib; {

